### DISCUSSION TITLE
Does kubeedge supports accessing service on edge from cloud

### DISCUSSION THREAD

**Discussion by stingshen** [kubeedge/kubeedge#2953] [Q&A] [answered]
I have some metric servers running on edge and prometheus on cloud needs to access it. 

The [edge mesh design](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/edgemesh-design.md) shows it supports `Service discovery on cloud`, but it doesn't seem to work on my environment.

Does Kubeedge support this scenario?

**Comment by kason-huang** [comment, marked as answer]
Edgemesh does not currently service discovery on cloud, and related work is underway. For more details, you can visit the https://github.com/kubeedge/edgeme. We will provide relevant proposals about Cross-Subnet Communication including Cross-Cloud Communication and Cross-Subnet Edge-Edge Communication.

**Reply by stingshen** [reply]
kubeedge 1.8 seems to implement the edge mesh design!
